strings/boyer-moore.py

```diff
@@ -0,0 +1,75 @@
+'''
+Boyer–Moore string-search algorithm is an efficient string-searching algorithm that is the standard benchmark for practical string-search literature.
+references: https://en.wikipedia.org/wiki/Boyer%E2%80%93Moore_string-search_algorithm
+'''
+class last_index(object):
+    # Generate a dictionary with the last index of each letter
+    def __init__(self, pattern, alphabet):
+        self.index = dict()
+        for letter in alphabet:
+            '''
+            rfind() method is returns the last index
```

Comment:
rfind() method returns the last index of the substring if found in given string. If not found then it returns -1.